src/ports/postgres/modules/convex/mlp_igd.py_in

```diff
@@ -1048,14 +1050,8 @@ def mlp_predict(schema_madlib, model_table, data_table, id_col_name,
         else:
             intermediate_col = unique_string()
             if classes:
-                # we cannot call sql quote_ident on the class value because
-                # aliasing does not support quote_ident. Hence calling our
-                # python implementation of quote_ident
-                score_format = ',\n'.join([
-                    'CAST({interim}[{j}] as DOUBLE PRECISION) as "{c_str}"'.
-                    format(j=i + 1, c_str=quote_ident("estimated_prob_{0}".
-                        format(str(c))).strip(' "'), interim=intermediate_col)
-                    for i, c in enumerate(classes)])
+                score_format = create_cols_from_array_sql_string(
+                    classes, intermediate_col, 'prob', 'double precision', False, 'convex') 
```

Comment:
For consistence, the module_name passed in be `MLP` instead of `convex`.

Reply:
done